### DISCUSSION TITLE
trigger CMAKE_SYSTEM_NAME STREQUAL "WindowsStore"

### DISCUSSION THREAD

**Discussion by foxnight2** [paullouisageneau/libdatachannel#1066] [Q&A]
I build the libdatachannel on windows and show the variable "CMAKE_SYSTEM_NAME".
The "CMAKE_SYSTEM_NAME " is `Windows`.

How can I trigger the `WindowsStore`?

Does it just do -DCMAKE_SYSTEM_NAME=WindowsStore ?
If you are on windows, you will not do -DCMAKE_SYSTEM_NAME=Linux.
The system will automatically detect the Windows.

**Comment by paullouisageneau** [comment]
You can't have CMake automatically guessing that you want to target Windows Store on Windows, so you need to pass `-DCMAKE_SYSTEM_NAME=WindowsStore`.

In the future, please ask CMake questions unrelated to libdatachannel like this one on a CMake forum instead of here, or refer directly to the CMake documentation.
